src/components/cards/DetailedCard.tsx

```diff
@@ -0,0 +1,78 @@
+import { Image } from 'components/Image';
+import { Typography } from 'components/typography/Typography';
```

Comment:
export all the components and import {Box, Image, Typography ...} from 'components';